index.js

```diff
@@ -2,10 +2,10 @@
  * Expose `pathToRegexp`.
  */
 module.exports = pathToRegexp
-module.exports.parse = parse
-module.exports.compile = compile
-module.exports.tokensToFunction = tokensToFunction
-module.exports.tokensToRegExp = tokensToRegExp
+pathToRegexp.parse = parse
+pathToRegexp.compile = compile
+pathToRegexp.tokensToFunction = tokensToFunction
+pathToRegexp.tokensToRegExp = tokensToRegExp
```

Comment:
Other arguments against this notwithstanding, in the esm version, these should be separate exports, not properties of the default export.

Reply:
Making named exports would be a breaking change.

But if necessary i can do it